routes/bookingRouter.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+
+const bookingRouter = express.Router();
+
+const bookingController = require("../controllers/bookingController");
+
+bookingRouter.get("/", bookingController.getcities);
```

Comment:
```suggestion
bookingRouter.get("/cities", bookingController.getcities);
```
자원을 명확히 표현해 주면 좋을 것 같습니다.